config/crd/bases/onload.amd.com_onloads.yaml

```diff
@@ -75,23 +65,20 @@ spec:
                       are heterogenous kernel versions on the nodes in the cluster.
                     items:
                       properties:
-                        build:
-                          description: Build contains build options for building kernel
-                            level components. If non-nil the operator will attempt
-                            to build the required images using what is defined in
-                            the build section, and push the images to the specified
-                            locations.
-                          type: object
                         kernelModuleImage:
-                          description: KernelImage is the image that contains the
-                            out-of-tree kernel modules used by onload, including the
-                            sfc driver.
+                          description: KernelModuleImage is the image that contains
+                            the out-of-tree kernel modules used by Onload.
                           type: string
                         regexp:
                           description: Regexp is a regular expression that is used
                             to match against the kernel versions of the nodes in the
                             cluster
                           type: string
+                        sfc:
+                          description: SFC specifies the driver image and various
+                            configuration knobs. If absent, the controller won't manage
+                            SFC in the given context.
+                          type: object
```

Comment:
True, there's no need for us to comment on the content of this yet-empty type.

I'm applying your suggestion nearly as-is:

1. In "api/v1alpha1/onload_types.go" instead of this YAML, because YAML is auto-generated.
2. SFC in all capitals because it comments on the struct field and that way it is aligned with the rest of the file.

Hope that's OK.